SECURITY.md

```diff
@@ -1,4 +1,6 @@
-# Security Policy
+# Security Reporting
```

Comment:
Should we maybe keep the old policy and add a new section as "Reporting" instead? I kinda find the previous page useful in terms of letting people know which versions of the project will support security updates.

Reply:
Agree 100%, and it is still in, look in the diff under "## Supported Versions"